docs/general/clients/jellyfin-vue.md

```diff
@@ -0,0 +1,227 @@
+    [major development milestones](https://jellyfin.org/posts/vue-vue3/).
+    * Only the latest `unstable` image is supported.
+
+After accessing the instance in your browser of choice, you'll be prompted to add a server. You can use your own server or our demo
+instance, located at `https://demo.jellyfin.org/stable`
+
+:::note
+The server address you need to type is relative to the device you're accessing Jellyfin Vue. For instance,
+if your Jellyfin Server is located at 192.168.0.10 and you have deployed Jellyfin Vue to that same server,
+and your client is 192.168.0.20, the address you need to input is `http://192.168.0.10`, not
+`http://127.0.0.1` or `http://localhost`.
+
+You can learn more about how the connection to your server works
+[in Jellyfin's Vue repository privacy disclaimer](https://github.com/jellyfin/jellyfin-vue#privacy-disclaimer-)
+:::
+
+### Using environment variables
+
+This is an example on how environment variables are set in the container by adding the
+following keys to the `jellyfin_vue` definition in [above's](#recommended-using-docker-compose) `docker-compose.yml`:
+
+```yaml
+environment:
+  HISTORY_ROUTER_MODE: 1
+```
+
+See the [complete reference of environment variables available](https://github.com/jellyfin/jellyfin-vue/wiki/Configuration)
+to further customize Jellyfin Vue to your liking at Jellyfin Vue's wiki.
+
+### Using your own webserver
+
+Since Jellyfin Vue is an SPA web application, you can use any webserver to serve it:
+Apache, nginx (the one used in Jellyfin's Vue docker image), Traefik, etc...
+
+If you already have a [reverse proxy](../networking/index.md) set up, do you want to do more complex stuff
+that it's out of the scope of this documentation like serving at a subpath, in another subdomain, etc...,
+you might want to have Jellyfin Vue served by your own webserver instead of the nginx instance shipped with
+the Docker image.
+
+To achieve that, in the `jellyfin_vue` service definition of the `docker-compose.yml` shown [above](#recommended-using-docker-compose):
+* Remove the `ports` key.
+* Add the following keys, **replacing** `_path_` **with the folder** where your webserver expects Jellyfin Vue's assets:
+
+```yaml
+network_mode: none
+volumes:
+  - _𝘱𝘢𝘵𝘩_:/dest
+# This makes the container do nothing and sleep forever,
+# frontend will be copied to _𝘱𝘢𝘵𝘩_ and will be served by your webserver
+# /setup.sh applies 
+entrypoint: /bin/sh -c '/setup.sh && rm -rf /dest/* && cp -r /usr/share/nginx/html/* /dest && sleep infinity'
+
+```
+
+This approach:
+* Keeps the client always updated in a hassle-free way, as explained [above](#recommended-using-docker-compose).
+* In case you use [environment variables](#using-environment-variables),
+they will still be applied.
+
+
+### Replacing Jellyfin Web
+
+:::danger
+Be aware that some clients, like Android (not TV) and Jellyfin Media Player expects that your Jellyfin server
```

Comment:
@iwalton3 I'm sure about Android, but not so about JMP. Is this accurate?

Reply:
This is correct